tests/TestCase/Mailer/MailerTest.php

```diff
@@ -0,0 +1,118 @@
+<?php
+namespace Cake\Test\TestCase\Mailer;
+
+use Cake\Mailer\Email;
+use Cake\Mailer\Mailer;
+use Cake\TestSuite\TestCase;
+
+class TestMailer extends Mailer
+{
+    public function getEmailForAssertion()
```

Comment:
Ok, we've had problems with rotting test stubs in the past which is why I personally try to avoid them.